### DISCUSSION TITLE
Welcome to NetScorpion Discussions!

### DISCUSSION THREAD

**Discussion by HPD1155** [HPD1155/NetScorpion#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions
  * Post ideas
  * Get support on NetScorpion

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prettycrunchyguy** [comment]
![image](https://github.com/HPD1155/NetScorpion/assets/137006173/cff0d170-9059-4a53-a14a-4c6016eaf89a)
